aws-iam-policies/domain-protect-deploy.json

```diff
@@ -38,14 +38,36 @@
       ],
       "Resource": "arn:aws:events:eu-west-1:SECURITY_AWS_ACCOUNT_ID:rule/domain-protect-*"
     },
+    {
+      "Effect": "Allow",
+      "Action": [
+          "iam:ListPolicies"
+      ],
+      "Resource": "*"
+    },
+    {
+        "Effect": "Allow",
+        "Action": [
+            "iam:AttachRolePolicy",
+            "iam:DetachRolePolicy",
+            "iam:GetPolicy",
+            "iam:GetPolicyVersion"
```

Comment:
Shouldn't getpolicy and getpolicyversion be in the block above?

Reply:
The least privilege approach is to limit to the policies which are actually needed.